src/Servers/Connections.Abstractions/src/IConnectionListenerFactory.cs

```diff
@@ -0,0 +1,17 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using System.Collections.Generic;
+using System.Net;
+using System.Text;
+using System.Threading;
+using System.Threading.Tasks;
+
+namespace Microsoft.AspNetCore.Connections
+{
+    public interface IConnectionListenerFactory
```

Comment:
How nice would it be to avoid the word factory? Maybe IEndpointBinder?

Reply:
Not changing the name in this PR, we agreed to this one in the meeting, we can iterate on the issue.

Reply:
That said, I don't disagree